code/go/0chain.net/core/util/mpt_nodedb.go

```diff
@@ -470,33 +519,33 @@ func (lndb *LevelNodeDB) Size(ctx context.Context) int64 {
 
 // PruneBelowVersion - implement interface.
 func (lndb *LevelNodeDB) PruneBelowVersion(ctx context.Context, version Sequence) error {
-	// TODO
+	// TODO: Why not implemented?
 	return nil
```

Comment:
I assume it is not implemented because it's not used anywhere. 

However, I don't like it silently failing; Either delete it or add in a panic. Unless there is a clear intention to use it later my preference is to delete it.